docs/products/clickhouse/concepts/disaster-recovery.md

```diff
@@ -52,16 +55,16 @@ only changed parts are backed up and files already available in the
 object storage are left out from the backup.
 :::
 
-### Recovery
-
-The restoration of a backup of an Aiven for ClickHouse service is
-performed on a running ClickHouse server and proceeds as a regular
-power-on of the service. The restoration happens only for powering up a
-service after powering it down or forking a service.
-
 For more information on backups in Aiven, see
 [Backups at Aiven](/docs/platform/concepts/service_backups).
 
+### Service recovery
+
+The [restoration of a backup](/docs/products/clickhouse/howto/restore-backup) of an
+Aiven for ClickHouse service is performed on a running ClickHouse server and proceeds as
```

Comment:
Fork is allowed even if the original clickhouse is powered off, Clickhouse server should not be running for that.